_pages/en/saas-and-cloud-strategy.md

```diff
@@ -0,0 +1,230 @@
+
+##### Leverage for Quick-Wins
+
+SaaS should be leveraged for 'quick wins' where only unprotected data is being used and for where it is not intended to contact citizens directly. Further, it should be leveraged when investment in an in-house developed solution would be in misalignment with responsible stewardship of public funds given the benefits said in-house delivery would deliver.
+
+> [Federal public servants are entrusted to use and care for public resources responsibly, for both the short term and long term.](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=25049)
+
+#### Future Actions
+
+##### Reduce Reliance on SaaS
+
+[TO-DO: DevOps Handbook Reference]
+
+SaaS can save time and money by not investing resources into the creation of existing tools.
+
+In-house development, due to heavy governance and approval processes, is a lengthy and thusly expensive process. However, as an organization
+
+## Conclusion
+
+### Answering the question: SaaS or Develop Cloud Based Solution?
+
+## References
+
+[Government of Canada Cloud Adoption Strategy: 2018 update](https://www.canada.ca/en/government/system/digital-government/modern-emerging-technologies/cloud-services/government-canada-cloud-adoption-strategy.html)
+
+[Digital Operations Strategic Plan: 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html)
+
+[2019 DORA State of DevOps Report](https://www.google.ca/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=2ahUKEwiN7Yfn5MTlAhWhnOAKHTdYDmkQFjABegQIAhAC&url=https%3A%2F%2Fcloudplatformonline.com%2Frs%2F248-TPC-286%2Fimages%2FDORA-State%2520of%2520DevOps.pdf&usg=AOvVaw2PLSLELi2td1WLLLoiqzcl)
+
+[Direction on the Secure Use of Commercial Cloud Services: Security Policy Implementation Notice (SPIN)](https://www.canada.ca/en/government/system/digital-government/modern-emerging-technologies/direction-secure-use-commercial-cloud-services-spin.html)
+
+[Direction for Electronic Data Residency](https://www.canada.ca/en/government/system/digital-government/modern-emerging-technologies/direction-electronic-data-residency.html)
+
+[Government of Canada Security Control Profile for Cloud-based GC Services](https://www.canada.ca/en/government/system/digital-government/modern-emerging-technologies/cloud-services/government-canada-security-control-profile-cloud-based-it-services.html)
+
+[IT Security Risk Management: A Lifecycle Approach (ITSG-33)](https://cyber.gc.ca/en/guidance/it-security-risk-management-lifecycle-approach-itsg-33)
+
+[Team Topologies](https://teamtopologies.com/)
+
+[US Federal Cloud Computing Strategy](https://cloud.cio.gov/strategy/)
+
+[Values and Ethics Code for the Public Sector](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=25049)
```

Comment:
Euh, What that used in the document above?
If not remove

But I'd see a definative fit in terms of an instrument for using SaaS